test/built-ins/Temporal/Duration/prototype/with/argument-singular-properties.js

```diff
@@ -1,20 +1,15 @@
-// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
We should add our copyright header as a second line.

Reply:
I wasn't sure about this, because I deleted the old file and created a new one, but I've added the V8 header now.